### ISSUE TITLE
Custom title search or Alternative title name

### ISSUE THREAD

**Issue by alexs20** [Sonarr/Sonarr#6233]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

I saw a lot of requests about either adding a custom search capability that will then link the search result to the release Title or adding an alternative Title to the release in order to allow better match between the release Title and actual list of sources found in the trackers. 
All these requests are usually being rejected with pointing to the fact that users can search manually in Prowlarr and then somehow feed that result into Sonarr.
But to be honest, I do not see any guide or document explaining how it is possible and how Sonarr will continue to monitor that release for new episodes and download them by itself.

Anyway, I think that either adding custom search capability on Sonarr level or allowing to add a custom alternative / alias Title is very useful for non-English language speaking users, many releases in trackers only have localized names but Sonarr uses names from TvDB as is, and they are 99.99% using latin keyset.
For example the show - "Kesha dolzhen umeret" which will never find any source, while there are plenty of sources for the correct name "Кеша должен умереть"

### Describe the solution you'd like

To add alternative title name with checkbox - "Use for searches"

### Describe alternatives you've considered

Custom search directly from added titile with option to reuse one or more results in future monitoring process

### Anything else?

I just hope that this recommendation will not be blindly dismissed and will be taken into consideration.

**Comment by alexs20** [comment]
I tried the manual importing approach, and it worked. 
However, as expected, if the show is in an ongoing state, Sonarr will not monitor new series. Therefore, the whole purpose of using Sonarr is nullified. I also attempted to submit corrections to the TVDB site, but the more titles I searched, the stronger my understanding became that almost 99% of Russian titles are invalid, requiring the submission of additional aliases. 
Now I understand why in Russian communities, people say that Sonarr is useless for Russian trackers. Nobody is going to spend extra time sending corrections to the shows repeatedly. The whole idea of Sonarr is to simplify life, not make it more complicated.
Therefore, thanks for the great tool, but it seems like it is not suitable for everybody yet due to the lack of flexibility.
